zombienet/tests/0007-full_node_warp_sync.toml

```diff
@@ -0,0 +1,77 @@
+[relaychain]
+default_image = "{{RELAY_IMAGE}}"
+default_command = "polkadot"
+default_args = [ "-lparachain=debug" ]
+default_db_snapshot = "https://github.com/samelamin/cumulus/blob/chain-snapshots/docker/snapshots/relaychain-1964f8b557f10085cdc18f4105ad0bbb3df4c4c6.tgz?raw=true"
```

Comment:
We need to remove this and add the db snapshots manually to the individual nodes. This is necessary because "dave" should not start with a full db but warp sync.